srt-protocol/src/protocol/pending_connection/hsv5.rs

```diff
@@ -33,13 +32,36 @@ pub fn gen_hsv5_response(
         }
     };
 
+    let mut acceptor = AllowAllStreamAcceptor::default();
```

Comment:
This is defintely one of the frustrating things about you merging your PRs together, is somtimes my comments go unaddressed...

No big deal, I know I could be faster about reviews and I don't have a problem saying it twice--anyways `accept_params` will always be Accept, so no need for this here

Reply:
> 

yeah, sorry, I thought I actually removed this.. hmmm. No worries on he speed of PRs, this is a labor of inspiration, no deadlines.